src/utils/components/PolicyForm/PolicyWizard/InterfacesStep.tsx

```diff
@@ -40,61 +36,55 @@ const InterfacesStep: FC<InterfacesStepProps> = ({ policy, setPolicy, error }) =
         };
       }
 
-      draftPolicy.spec.desiredState.interfaces.unshift({
-        type: InterfaceType.LINUX_BRIDGE,
-        name: `interface-${draftPolicy.spec.desiredState.interfaces.length}`,
+      const newInterface: NodeNetworkConfigurationInterface = {
+        type: interfaceTypes?.[0],
```

Comment:
You use `interfaceTypes?.[0]` multiple places in this file. Consider storing it in a variable.